hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/TreeLockManager.java

```diff
@@ -231,25 +231,27 @@ private boolean retryBackoff(int retries) throws IOException {
    * or write locks should be held by anyone on any child directory.
    *
    * @param path Path to lock
+   * @param depth Depth.DIRECTORY to look for locks on this path only, or Depth.RECURSIVE to
```

Comment:
Would it be better to use a Javadoc link for the `Depth` parts?

Reply:
Thanks for reviewing it. Yes, and while checking on this, I realised there are 25 more javadoc warnings. Will work on all of them and push in the next commit.